_field-types/mapping-parameters/analyzer.md

```diff
@@ -2,89 +2,207 @@
 
-The key functions of the `analyzer` mapping parameter are:
+Only `text` fields support the `analyzer` mapping parameter.
+{: .important}
 
-1. **Tokenization:** The analyzer determines how the text is broken down into individual tokens (words, numbers) that can be indexed and searched. Each generated token must not exceed 32,766 bytes in order to avoid indexing failures.
+The `analyzer` parameter cannot be updated on existing fields using the Update Mapping API. To change the analyzer for an existing field, you must reindex your data.
+{: .warning}
 
-2. **Normalization:** The analyzer can apply various normalization techniques, such as converting text to lowercase, removing stop words, and stemming/lemmatizing words.
+We recommend testing analyzers before deploying them to production environments.
+{: .tip}
 
-3. **Consistency:** By defining the same analyzer for both index and search operations, you ensure that the text analysis process is consistent, which helps improve the relevance of search results.
+## Search quote analyzer
 
-4. **Customization:** OpenSearch allows you to define custom analyzers by specifying the tokenizer, character filters, and token filters to be used. This gives you fine-grained control over the text analysis process.
+The `search_quote_analyzer` parameter allows you to specify a different analyzer specifically for phrase queries. This proves especially valuable when you need to handle stop words differently for phrase searches versus regular term searches.
 
-For information about specific analyzer parameters, such as `analyzer`, `search_analyzer`, or `search_quote_analyzer`, see [Search analyzers]({{site.url}}{{site.baseurl}}/analyzers/search-analyzers/).
-{: .note}
+For effective phrase query handling with stop words, configure three analyzer settings:
 
-------------
+1. An `analyzer` for indexing that preserves all terms, including stop words.
+2. A `search_analyzer` for regular queries that filters out stop words.
+3. A `search_quote_analyzer` for phrase queries that retains stop words.
 
 ## Example
 
-The following example configuration defines a custom analyzer called `my_custom_analyzer`:
+The following example demonstrates how to use `search_quote_analyzer` to handle stop words differently in phrase queries versus term queries.
+
+First, create an index with all three analyzer types. The `index_analyzer` preserves all terms during indexing, including stop words like "the" and "a". The `search_analyzer` removes stop words from regular term queries. The `search_quote_analyzer` uses the same analyzer as indexing for phrase queries, ensuring exact phrase matching works correctly:
```

Comment:
Last sentence: "uses the same analyzer as indexing for phrase queries" => "uses the same analyzer used for phrase query indexing"?